generator/lib/behavior/i18n/I18nBehaviorObjectBuilderModifier.php

```diff
@@ -183,7 +183,7 @@ protected function addTranslatedColumnSetter(Column $column)
             $objectBuilder->addMutatorOpenOpen($functionStatement, $column);
         }
         $comment = preg_replace('/^\t/m', '', $comment);
-        $comment = str_replace('@return     ' . $i18nTablePhpName, '@return     ' . $tablePhpName, $comment);
+        $comment = str_replace('@return ' . $i18nTablePhpName, '@return ' . $tablePhpName, $comment);
         $functionStatement = preg_replace('/^\t/m', '', $functionStatement);
```

Comment:
the origin code we are replacing here is also generated, isn't it? If so, we can safely assume how it looks like and adjust the $replace to whatever we want it to be...